### PR TITLE
fixes when testing in 3.11

### PR DESCRIPTION
**Pull request by martindemello**

### CODE REVIEW

File: tests/testdata/3.10/flow.cpython-310.pyc
Comment (rchen152) [inline tests/testdata/3.10/flow.cpython-310.pyc:1]: Are these pyc files supposed to be checked in?

File: pycnite/mapping.py [hunk 1]
```diff
@@ -352,5 +352,18 @@ def get_mapping(version: Tuple[int, int]) -> OpMap:
 }
 
 
-def arg_type(name: str):
-    return ARG_TYPES.get(name)
+PYTHON_3_11_ARG_TYPES: OpArgs = PYTHON_3_8_ARG_TYPES.copy()
```
Comment (rchen152) [inline pycnite/mapping.py]: optional nit:
```
some_dict = {**other_dict, 'extra': 'key'}
```
would be a little more concise

File: pycnite/__version__.py [hunk 2]
```diff
@@ -1,2 +1,2 @@
 # pylint: skip-file
-__version__ = "2023.09.08"
+__version__ = "2023.09.13"
```
Comment (rchen152) [inline pycnite/__version__.py]: Should probably be 2023.09.14 now.

File: pycnite/mapping.py (hunk 1 above)
Comment (martindemello) [reply pycnite/mapping.py]: done

File: pycnite/__version__.py (hunk 2 above)
Comment (martindemello) [reply pycnite/__version__.py]: done

File: tests/testdata/3.10/flow.cpython-310.pyc
Comment (martindemello) [reply tests/testdata/3.10/flow.cpython-310.pyc:1]: yeah, the pyc files are the actual testdata. saves developers having to install pyenv and a bunch of versions unless they want to generate new test files.

**Review by rchen152 — APPROVED** [review]

**Pull request merged**